tests/models/audio_spectrogram_transformer/test_feature_extraction_audio_spectrogram_transformer.py

```diff
@@ -136,6 +143,117 @@ def test_call(self):
         for enc_seq_1, enc_seq_2 in zip(encoded_sequences_1, encoded_sequences_2):
             self.assertTrue(np.allclose(enc_seq_1, enc_seq_2, atol=1e-3))
 
+    def test_call_with_time_mask_length_and_frequency_mask_length(self):
```

Comment:
let's add a test that just ensures that you can do a forward pass when doing time + frequency masking with temporal shift and random noise !